internal/core/services/scheduler_manager/scheduler_manager.go

```diff
@@ -140,6 +141,27 @@ func (s *SchedulerManager) CreateNewSchedulerVersionAndEnqueueSwitchVersion(ctx
 	return nil
 }
 
+func (s *SchedulerManager) PatchSchedulerAndSwitchActiveVersionOperation(ctx context.Context, schedulerName string, patchMap map[string]interface{}) (*operation.Operation, error) {
```

Comment:
This method name is kind of strange because it enqueues `CreateNewSchedulerVersion`, and not the `SwitchActiveVersion`... What do you think about renaming to `PatchSchedulerAndCreateNewSchedulerVersion`?

Reply:
Looks great!